### PR TITLE
Adds inputs to README

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #117

## Description

Refs: #issue_number

* * *

## Before you get started

- [ ] I read and I followed [contributing guidelines](https://github.com/tortellini-tools/action/blob/main/CONTRIBUTING.md)
- [ ] [☝ Create an issue](https://github.com/NLeSC/licenseguard/issues/new/choose) to discuss what you are going to do

## Checklist

This section should be filled in by the creator of this pull request to make sure the pull request is ready to review.

- [ ] This pull request has a descriptive title
- [ ] Code is written according to the code quality guidelines
- [ ] Documentation if applicable is available
- [ ] Tests run successfully
- [ ] All checks below this pull request were successful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,79 @@
...
-## Usage
+**Required** A file or URL containing [ort rules](https://github.com/oss-review-toolkit/ort/blob/master/docs/file-rules-kts.md) to detect license violations. Default is [https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts](https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts).
+
+### `classifications`
+
+**Required** A file or URL containing classes for each license. For format see [ort documentation](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-license-classifications-yml.md). Default is [https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts](https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts).
+
+### `curations`
+
+A file or URL containing curations correct invalid or missing package metadata and set the concluded license for packages. See [ort documentation](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-curations-yml.md) for format.
```
Comment (fdiblen) [inline README.md]: Not clear if this is required or optional. Can you add that?

File: README.md [hunk 2]
```diff
@@ -1,62 +1,79 @@
...
 
-See [action.yml](action.yml)
+## Outputs
+
+No action outputs are defined for this actions.
+However the action will write files to `.tortellini/out/` directory.
```
Comment (fdiblen) [inline README.md]:
```suggestion
The action will write files to `.tortellini/out/` directory by default.
```

File: README.md [hunk 3]
```diff
@@ -1,62 +1,79 @@
...
+    tortellini:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - run: |
+                  echo 'https://github.com/tortellini-tools/action' > urls.txt
+                  echo 'https://github.com/fair-software/howfairis' >> urls.txt
+            - uses: tortellini-tools/action@main
+              with:
+                  repositories: 'urls.txt'
```
Comment (fdiblen) [inline README.md]:
```suggestion
                  repositories: urls.txt
```

File: README.md [hunk 4]
```diff
@@ -1,62 +1,79 @@
...
-        curations: .tortellini.yml
-        rules: https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts
-        classifications: https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/license-classifications.yml
+    tortellini:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: tortellini-tools/action@main
+              with:
+                  curations: .tortellini.yml
```
Comment (fdiblen) [inline README.md]:
```suggestion
                  curations: curations.yml
```

**Review by fdiblen — CHANGES_REQUESTED** [review]
Thank you @sverhoeven. I made a few recommendations.

File: README.md (hunk 2 above)
Comment (sverhoeven) [reply README.md]: If you say `by default` I expect that there is a way to changed which is not the case. See #85. I left out that part

File: README.md (hunk 1 above)
Comment (sverhoeven) [reply README.md]: done

**Comment by sverhoeven** [comment]
Thanks for reviewing.
I incorporated some of your suggestions and others became outdated after merging of #136 .

Could you have another look?

**Review by fdiblen — APPROVED** [review]
LGTM. 👍

**Pull request merged**